pom.xml

```diff
@@ -157,6 +173,14 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>license-maven-plugin</artifactId>
+        <configuration>
+          <licenseName>bsd_2</licenseName>
+          <projectName>OME-Zarr Bio-Formats reader</projectName>
```

Comment:
Based on https://github.com/ome/bioformats/blob/develop/components/formats-bsd/pom.xml#L233 perhaps something like:

- Zarr-based implementations of Bio-Formats readers and writers

Reply:
or if the repo tends towards formats-ngff:

 * Next-generation implementations ...

Reply:
k